src/modules/rtpengine/rtpengine.c

```diff
@@ -1675,6 +1677,20 @@ static int build_rtpp_socks() {
 				LM_WARN("Failed enable set MTU discovery socket option\n");
 #endif
 
+			if((0 <= control_cmd_tos) && (control_cmd_tos < 256)) {
+				unsigned char tos = control_cmd_tos;
+				if (pnode->rn_umode == 6) {
+					setsockopt(rtpp_socks[pnode->idx], IPPROTO_IPV6,
+							IPV6_TCLASS, &tos,
+							sizeof(tos));
```

Comment:
Sorry, I guess I should have been more clear. The argument to the IPv6 version IPV6_TCLASS is in fact an int. It's just the IPv4 IP_TOS that takes an unsigned char.

Reply:
I have seen in the following link (https://notes.shichao.io/unp/ch7/) that the data type for IP_TOS is "int". Therefore i also changed for IPv4, otherwise your comment was clear. Anyway, i have changed it based on your review again, thank you for your time.